api-server/config/openapi.yml

```diff
@@ -298,6 +298,69 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/BrokersResponse'
+  /readAddressAttributes:
+    get:
+      summary: read address attributes
+      description: >
+        **Read values of address attributes**
+          The return value is a json array that contains
+          values of requested attributes of the addresses's mbean.
+
+          **Example:**
+          
+          **Request url:** https://localhost:9443/api/v1/readAddressAttributes?name=DLQ,attrs=RoutingTypes
+          (To read the `RoutingTypes` attribute of the address DLQ)
+
+          **Response:**
+          ```json
+          [
+            {
+              "request": {
+                "mbean": "org.apache.activemq.artemis:address=\"DLQ\",broker=\"amq-broker\",component=addresses",
+                "attribute": "RoutingTypes",
+                "type": "read"
+              },
+              "value": [
+                "ANYCAST"
+              ],
+              "timestamp": 1715864988,
+              "status": 200
+            }
+          ]
+          ```
+          **Note**: to read multiple attributes, set it to **attrs** parameter
+          separated by commas, e.g. `RoutingTypes,Address`.
+      tags:
+        - jolokia
+      operationId: readAddressAttributes
+      parameters:
+        - in: header
+          name: jolokia-session-id
+          schema:
+            type: string
+          required: true
+        - name: name
+          schema:
+            type: string
+          required: true
+          in: query
+        - name: attrs
+          description: attribute names separated by commas. If not speified read all attributes.
+          required: false
+          in: query
+          schema:
+            type: array
+            items:
+              type: string
+          style: form
+          explode: false
+      responses:
+        200:
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BrokersResponse'
```

Comment:
Having a more detailed schema will make the code generation do the Typescript work for us:

```
      responses:
        200:
          description: Success
          content:
            application/json:
              schema:             
                type: array
                items:
                  $ref: '#/components/schemas/AddressAttribute'
```

with at the end of the document a new `AddressAttribute` definition:

```
    AddressAttribute:
      type: object
      properties:
        request:
          type: object
          properties:
            mbean:
              type: string
            attribute:
              type: string
            type:
              type: string
        value:
          type: array
          items:
            type: string
        timestamp:
          type: number
        status:
          type: number
```

Reply:
That's nice to have. Currently `BrokerResponse` is used as a common response schema for almost every api. I think we can do this for all other apis not only for this api. I've create a issue for that:
https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/issues/146

Reply:
This PR is adding a new endpoint, it makes sense to have it already returning a tailored result, #140 would greatly benefit of this tailoring.

I agree that a followup PR is necessary for all the other endpoints in any case.

Reply:
https://github.com/artemiscloud/activemq-artemis-self-provisioning-plugin/issues/146 is a common issue, and it should be solved separately. I would suggest not to block this PR by it. The major purpose of this PR is add a new endpoint. Not to partially solve another issue.